examples/middleware.rs

```diff
@@ -19,11 +19,14 @@ impl<C: HttpClient> Middleware<C> for Printer {
     }
 }
 
-#[runtime::main]
-async fn main() -> Result<(), surf::Exception> {
-    femme::start(log::LevelFilter::Info)?;
-    surf::get("https://httpbin.org/get")
-        .middleware(Printer {})
-        .await?;
-    Ok(())
+use async_std::task;
+
+fn main() {
```

Comment:
This should probably return:
```suggestion
fn main() -> Result<(), surf::Exception> {
```

so that `unwrap` isn't needed.